### PR TITLE
Makes the plugin package discoverable

### PR DESCRIPTION
**Pull request by Voldivh**

### CODE REVIEW

**Review by francocipollone — COMMENTED** [review]
Before diving into review
1. Did you find which was the cause of the plugin not being discoverable?
2. Note that there are some changes that I think aren't related to the PR's purpose(at least pr's title.)
   - Linters being added
   - resources being added.

**Comment by Voldivh** [comment]
> 1. Did you find which was the cause of the plugin not being discoverable?

Yes, it appers there were 2 simple things that caused troubles:
1. On the package.xml we defined the licence by doing `<license file='LICENSE'> BSD 3 Clause </license>` and when I was testing the package it gave me problem the `file='LICENSE'` so I removed it.
2. I noticed that on the setup.cfg the `[develop]` and `[install]` were using snake_case ('_') instead of hyphen ('-') on the naming of the variables (I'm not sure this actually makes a difference but I left it with hyphen since that is how it is created when doing `ros2 pkg create --build-type ament_python <package_name>`).

> 2. Note that there are some changes that I think aren't related to the PR's purpose(at least pr's title.)
>    
>    * Linters being added
>    * resources being added.
You are right, I forgot to delete a couple of things. For the resource I changed the name of the .ui file to keep consistency with other rqt plugins.

File: package.xml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+<package format="3">
   <name>rqt_gauges_2</name>
   <version>0.0.1</version>
   <description>Visualization plugin for several sensors.</description>
+  
+  <maintainer email="eloy.briceno@ekumenlabs.com">Eloy Bricneo</maintainer>
 
-  <maintainer email="eloy.briceno@ekumenlabs.com">Eloy Briceno</maintainer>
-
-  <license file="LICENSE">BSD Clause 3</license>
```
Comment (francocipollone) [inline package.xml:9]: Revert this change, It shouldn't be related.

PS: I tested locally just in case and it doesn't seem to affect.

File: resource/gauges_2.ui [hunk 2]
```diff
@@ -29,4 +29,4 @@
  </widget>
  <resources/>
  <connections/>
-</ui>
+</ui>
```
Comment (francocipollone) [inline resource/gauges_2.ui]: Small unintended change

File: setup.py [hunk 3]
```diff
@@ -11,16 +11,19 @@
...
     ],
     install_requires=['setuptools'],
+    zip_safe=True,
     maintainer='Eloy Briceno',
     maintainer_email='eloy.briceno@ekumenlabs.com',
     description=(
         'rqt_gauges_2 is a Python GUI plugin providing a visualization tool for several sensors.'
     ),
+    license='BSD Clause 3',
+    tests_require=['pytest'],
```
Comment (francocipollone) [inline setup.py]: Don't belongs to this PR.

File: setup.py [hunk 4]
```diff
@@ -11,16 +11,19 @@
         ('share/ament_index/resource_index/packages',
             ['resource/' + package_name]),
         ('share/' + package_name + '/resource',
-            ['resource/gauges_2_widget.ui']),
+            ['resource/gauges_2.ui']),
         ('share/' + package_name, ['package.xml']),
         ('share/' + package_name, ['plugin.xml']),
     ],
     install_requires=['setuptools'],
+    zip_safe=True,
```
Comment (francocipollone) [inline setup.py]: What is this for?

**Review by francocipollone — CHANGES_REQUESTED** [review]
LGTM, just some comments

File: setup.py (hunk 4 above)
Comment (Voldivh) [reply setup.py]: I'm not sure, it came as default with the command `ros2 pkg create --build-type ament_python <package_name>` so i left it.

File: resource/gauges_2.ui (hunk 2 above)
Comment (Voldivh) [reply resource/gauges_2.ui]: So there is no change to be done from what I understand.

Comment (francocipollone) [reply resource/gauges_2.ui]: Check the tab where you review the pr
image

File: setup.py (hunk 4 above)
Comment (francocipollone) [reply setup.py]: https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html
Just delete it

File: resource/gauges_2.ui (hunk 2 above)
Comment (Voldivh) [reply resource/gauges_2.ui]: Oh I see.

File: setup.py (hunk 4 above)
Comment (Voldivh) [reply setup.py]: Makes sense. Done.

File: setup.py (hunk 3 above)
Comment (Voldivh) [reply setup.py]: Done.

File: package.xml (hunk 1 above)
Comment (Voldivh) [reply package.xml:9]: Done.

**Review by francocipollone — APPROVED** [review]
LGTM
